sdk/storage/Azure.Storage.Blobs/README.md

```diff
@@ -0,0 +1,115 @@
+# Azure Storage Blobs client library for .NET
+Azure Blob storage is Microsoft's object storage solution for the cloud. Blob
+storage is optimized for storing massive amounts of unstructured data.
+Unstructured data is data that does not adhere to a particular data model or
+definition, such as text or binary data.
+
+[Source code][source] | [Package (NuGet)][package] | [API reference documentation][rest_docs] | [Product documentation][product_docs]
+
+## Getting started
+### Install the package
+Install the Azure Storage Blobs client library for .NET with [NuGet][nuget]:
+
+```Powershell
+Install-Package Azure.Storage.Blobs -Version 1.0.0-preview.1
+```

```

Comment:
Does markdown have any feature to make a constant out of the version number?

Reply:
I copied this from the Config service example, but I'll yank the versions because they're not required and then it won't ever need updating.